requirements.txt

```diff
@@ -1,4 +0,0 @@
-requests==2.22.0
-siggen<2
-ujson==1.35
-google-cloud-bigquery==1.27.2
```

Comment:
Because fx-crash-sig is both a library on PyPI and also (now) an etl job, I moved the requirements to be in `setup.py` which gets run in both scenarios.